config/locales/activerecord.fr.yml

```
@@ -88,6 +89,9 @@ fr:
         delegue_protection_donnees_phone_number: Téléphone du délégué à la protection des données
         delegue_protection_donnees_job_title: Fonction du délégué à la protection des données
 
+        extra_organization_contact_name: Nom de l'organisation
```

Je suis brainfuck par ce truc, de quoi on parle ici ? Cette info on l'a déjà. Je suis allé voir les maquettes et en effet c'est écrit ça mais c'est évidemment une typo.

Je mettrais ici d'ailleurs:

```suggestion
        extra_organization_contact_name: Nom du contact supplémentaire
```

le nom de l'attribut ne me parait clair btw

---

peut-être plus directement "extra_contact_*" ou "additional_contact_*" ?

---

C'est étrange le `extra` seul : extra de quoi ? ça me semble logique que ça soit lié à l'orga

---

est-ce qu'il manque un "pas" ou est-ce qu'il y a un "ne" en trop dans "le nom de l'attribut ne me parait clair btw" ? :smile:

---

mais ouais j'ai traduit "contact supplémentaire" assez littéralement, je suis toujours preneur d'un meilleur naming

---

> est-ce qu'il manque un "pas" ou est-ce qu'il y a un "ne" en trop dans "le nom de l'attribut ne me parait clair btw" ? 😄

il manque un pas